csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -150,9 +150,8 @@ function setUpMode(){
   }
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/58441963-b36f1a80-813a-11e9-8ae7-3e238ebe4b85.png)
The apply blur button will change the image to greyscale regardless of the setting you chose